### PR TITLE
Fix test: RegistryEntryXsltTransformationTestCase

### PR DESCRIPTION
**Pull request by RidmiR**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform.xslt [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (GDLMadushanka) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform.xslt]: 2019

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform_back.xslt [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (GDLMadushanka) [inline integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform_back.xslt]: 2019

Comment (RidmiR) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform_back.xslt]: Changed. Please refer the latest commit.

File: integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform.xslt (hunk 1 above)
Comment (RidmiR) [reply integration/mediation-tests/tests-mediator-2/src/test/resources/artifacts/ESB/server/registry/governance/xslt/transform.xslt]: Changed. Please refer the latest commit.

**Pull request merged**